frontend/src/scenes/web-analytics/WebDashboard.tsx

```diff
@@ -275,12 +280,12 @@ export const WebTabs = ({
 }
 
 export interface LearnMorePopoverProps {
-    docsURL: PostHogComDocsURL
+    url?: PostHogComResourcesURL
```

Comment:
did this change get used in the end? I don't see us using tutorial links anywhere?

the reason that docs links are special is because the open the sidebar rather than being an external link

Reply:
Removed the change